lightyear/Cargo.toml

```diff
@@ -30,7 +30,7 @@ webtransport = [
   "dep:tokio",
   "dep:ring",
 ]
-leafwing = ["dep:leafwing-input-manager", "lightyear_macros/leafwing"]
+leafwing = ["dep:leafwing-input-manager", "lightyear_macros/leafwing", "bevy/bevy_asset"]
```

Comment:
is bevy asset necessary for leafwing?

Reply:
It was necessary for the tests to run in the Leafwing feature (for use bevy::asset::AsyncReadExt), but I think it's not necessary for the build. I'm not entirely sure how to convince Cargo to only include a dependency for test with a certain feature though.